content/04-postgresql/07-create-and-delete-databases-and-tables.mdx

```diff
@@ -561,3 +561,63 @@ This will remove the database and any child elements, like tables, within.
 This article covered the basics of how to create and delete databases and tables within PostgreSQL.  These are some of the most basic commands required to set up a database system and being defining the structure of your data.
 
 As mentioned earlier, the SQL statements covered in this PostgreSQL tutorial, particularly the `CREATE TABLE` statement, have many additional parameters can be used to change PostgreSQL's behavior.  You can find out more about these by checking out the [official PostgreSQL documentation](https://www.postgresql.org/docs/current/sql-createtable.html).
+
+## FAQ
+
+
+<details><summary>Does PostgreSQL support the `IF NOT EXISTS` clause when using the `CREATE DATABASE` command?</summary>
+
+Yes, PostgreSQL supports the use of `IF NOT EXISTS` when creating both databases and tables. The below demonstrates using the clause for table creation, and you can read more about its use [here](https://www.prisma.io/dataguide/postgresql/create-and-delete-databases-and-tables#create-tables-conditionally-with-the-if-not-exists-clause) 
+
+```sql
+CREATE TABLE IF NOT EXISTS table_name (
+    column_name TYPE [column_constraint],
+    [table_constraint,]
+);
+```
+
+</details>
+
+<details><summary>How do you create a database from a dump in PostgreSQL?</summary>
+
+To create a database from a dump, PostgreSQL provides the [utility program `pg_dump`](https://www.postgresql.org/docs/9.1/backup-dump.html).
+
+This program recreates the database in the same state as it was at the time of the dump. Example syntax would look like the following:
+
+```sql
+pg_dump dbname > outfile
+```
+
+</details>
+
+<details><summary>Which command line creates a database in PostrgreSQL?</summary>
```

Comment:
I'd double check with the intention behind the "command line" part here only because PostgreSQL installations also include a command line utility called `createdb`.  If they are wanting a description of using that, you can find it [in the docs](https://www.postgresql.org/docs/current/app-createdb.html) but it basically is just typing this from your shell instead of within the `psql` client:

```bash
createdb db_name
```